plugin/pkg/scheduler/algorithm/priorities/interpod_affinity.go

```diff
@@ -98,99 +87,85 @@ func (ipa *InterPodAffinity) CalculateInterPodAffinityPriority(pod *api.Pod, nod
 	}
 
 	// convert the topology key based weights to the node name based weights
-	var maxCount int
-	var minCount int
-	counts := map[string]int{}
-	for _, node := range nodes {
-		totalCount := 0
-		// count weights for the weighted pod affinity
-		if affinity.PodAffinity != nil {
-			for _, weightedTerm := range affinity.PodAffinity.PreferredDuringSchedulingIgnoredDuringExecution {
-				weightedCount, err := ipa.CountWeightByPodMatchAffinityTerm(pod, allPods, weightedTerm.Weight, weightedTerm.PodAffinityTerm, node)
-				if err != nil {
-					return nil, err
+	var maxCount float64
+	var minCount float64
+	counts := make(map[string]float64, len(nodes))
```

Comment:
write a comment explaining what the map key and value represent (I realize it's the same thing as previously)